schema.sql

```diff
@@ -1,11 +1,29 @@
 /* Database schema to keep the structure of entire database. */
 CREATE TABLE animals (
-	user_id integer SERIAL PRIMARY KEY,
+	id integer SERIAL PRIMARY KEY,
 	name VARCHAR,
 	date_of_birth date,
 	escape_attempts integer,
 	neutered bool,
 	weight_kg DECIMAL
 );
 
-ALTER TABLE animals ADD species VARCHAR;
\ No newline at end of file
+ALTER TABLE animals ADD species VARCHAR;
+
+-- Start Day 3 Tasks
+
+CREATE TABLE owners (
+	id SERIAL PRIMARY KEY,
+	full_name VARCHAR,
+	age integer
+);
+
+CREATE TABLE species (
+	id SERIAL PRIMARY KEY,
+	name VARCHAR
+);
+
+alter table animals add column species_id INTEGER REFERENCES animals (id)
```

Comment:
- [ ] I think the `species_id` column in the animals' table **should be a foreign key REFERENCING id column in the species table.** What you have done is that it will reference the id column in the animals' table.
Please correct that by running this command 👇 
```sql
ALTER TABLE animals ADD species_id int REFERENCES species(id);
```